frontend/src/components/opportunity/OpportunityDescription.tsx

```diff
@@ -53,14 +53,19 @@ const OpportunityDescription = ({ opportunityData }: Props) => {
     .applicant_eligibility_description
     ? opportunityData.summary.applicant_eligibility_description
     : "--";
+  const agency_email = opportunityData.summary.agency_email_address
```

Comment:
seems like we can use nullish coalescing (`??`) in all of these defaulting situations, slightly legible, less verbose

Reply:
I updated these, thanks.